features/flows/rest/impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,11 +15,19 @@
     <reference id="transactionOperations" interface="org.springframework.transaction.support.TransactionOperations" availability="mandatory"/>
     <reference id="classificationService" interface="org.opennms.netmgt.flows.classification.ClassificationService" availability="mandatory"/>
 
+  <!-- Configuration properties -->
+	<cm:property-placeholder id="flowGraphUrlProperties" persistent-id="org.opennms.netmgt.flows.rest.internal" update-strategy="reload">
```

Comment:
Can we drop the '.internal' here and change the pid to `org.opennms.netmgt.flows.rest` instead?